src/index.json

```diff
@@ -720,8 +720,8 @@
         ],
         "aks-preview": [
             {
-                "downloadUrl": "https://azurecliaks.blob.core.windows.net/azure-cli-extension/aks_preview-0.4.35-py2.py3-none-any.whl",
-                "filename": "aks_preview-0.4.35-py2.py3-none-any.whl",
+                "downloadUrl": "https://azcliprod.blob.core.windows.net/cli-extensions/aks_preview-0.5.9-py2.py3-none-any.whl",
```

Comment:
What if the users insist to use old version?

Reply:
Ask them to install with `--source` as the file is still there.